packages/opentelemetry-core/src/context/propagation/B3Propagator.ts

```diff
@@ -46,17 +54,26 @@ export class B3Propagator implements HttpTextPropagator {
   inject(context: Context, carrier: unknown, setter: SetterFunction) {
     const spanContext = getParentSpanContext(context);
     if (!spanContext) return;
-
+    const parentSpanId = context.getValue(PARENT_SPAN_ID_KEY);
     if (
       isValidTraceId(spanContext.traceId) &&
       isValidSpanId(spanContext.spanId)
     ) {
+      if (parentSpanId) {
+        if (isValidTraceId(parentSpanId as string))
```

Comment:
Not sure if there is a linting rule for it, but we prefer to always use brackets in if blocks.